docs/04-operation-guides/operations/evnt-02-eventing-metrics.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Kyma Eventing Metrics
+---
+
+Kyma Eventing emits several custom metrics which are then visualized using [Grafana Dashboards](./evnt-01-eventing-dashboards.md) to monitor statistics and other information for the Eventing backbone in real time.
+
+### Metrics Emitted by Eventing Publisher Proxy:
+
+| Metric    |  Description |
+|-------------|:--------------|
+| **event_publish_to_messaging_server_errors_total** | Total number of errors while sending events to the messaging server. |
+| **event_publish_to_messaging_server_latency** | Duration of sending events to the messaging server. |
+| **event_type_published** | Total number of events published for a given event type. |
+| **event_requests** | Total number of event requests for a destination service.  |
+
+### Metrics Emitted by Eventing Controller:
+
+| Metric    |  Description |
+|-------------|:--------------|
+| **event_type_subscribed** | All the eventTypes subscribed using the Subscription CRD. |
+| **delivery_per_subscription** | Number of dispatched events per subscription with information regarding the status code and its sink. |
+
+### Metrics Emitted by NATS Exporter:
```

Comment:
Should we document the stuff which is not implemented by us? 
IMO, we can just add a reference to NATS documentation for exposed metrics. Otherwise, we would need to maintain this section every-time NATS changes something.

Reply:
The reason I added this was because I did not see any of this information (exposed metrics) documented by NATS. The only way for me to search for the metric name was to check the `/metrics` endpoint. The ones I mentioned here are mostly the ones we use in our dashboard

Reply:
yes, this is for user. IMO, we should only communicate the metrics which we are exposing as part of our Eventing services. If some user is interested in NATS metrics, they can see the NATS documentation: https://docs.nats.io/running-a-nats-service/nats_admin/monitoring